integrations/outlook/syncs/emails.ts

```
@@ -51,7 +59,18 @@ function extractHeaders(message: OutlookMessage): Record<string, any> {
     };
 }
 
-function processParts(attachments: Attachment[]): Attachments[] {
+async function fetchApiAttachments(nango: NangoSync, messageId: string): Promise<Attachment[]> {
+    const config: ProxyConfiguration = {
+        endpoint: `/v1.0/me/messages/${messageId}/attachments`,
```

ESLint

[error] Endpoint properties should include a documentation URL in a comment above

(@nangohq/custom-integrations-linting/include-docs-for-endpoints)